framework/packages/abstract-standalone/src/endpoints/instantiate.rs

```diff
@@ -17,7 +16,7 @@ impl StandaloneContract {
     pub fn instantiate(
```

Comment:
Can you add a doc comment to ensure users know the info.sender must be the module factory?

```
/// Instantiates the `Standalone` state for this contract. 
/// **Note:** This contract can only be instantiated by the abstract module factory. 
```